datastore/elastic_search/query.py

```diff
@@ -388,7 +399,7 @@ def _generate_es_search_dictionary(entity_name, text, fuzziness_threshold, langu
                 'should': [],
                 'minimum_should_match': 1
             }
-        }
+        }, 'from': 0, 'size': constants.ELASTICSEARCH_SEARCH_SIZE
```

Comment:
remove `'from': 0` as it defaults to 0 anyway

Reply:
Done